products/node-feature-discovery.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Kubernetes Node Feature Discovery
+category: server-app
+tags: kubernetes
+iconSlug: kubernetes
+permalink: /kubernetes-node-feature-discovery
+alternate_urls:
+-   /k8s-node-feature-discovery
+-   /node-feature-discovery
+versionCommand: |-
+  # Must be installed through helm chart https://artifacthub.io/packages/helm/node-feature-discovery/node-feature-discovery
+  nfd-gc -version
+  nfd-master -version
+  nfd-topology-updater -version
+  nfd-worker -version
+releasePolicyLink: https://github.com/kubernetes-sigs/node-feature-discovery/releases
```

Comment:
This doesn't provide any policy information. The [Security Policy](https://github.com/kubernetes-sigs/node-feature-discovery?tab=security-ov-file#readme) also doesn't mention any clear support policy, only k8s-compatibility policy, which isn't enough for here.

Do we have a better link?